### ISSUE TITLE
[AllyAudit] Add focus indication to Select and Error Inputs

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1323]
**User Story**
As Spark we want to add focus indication to Select Input and Error Input so that there is an indication other than the color change for assistive technologies. 

**Notes:**
Paciello Audit on Spark Design System
QLC-22 INPUTS - Select and Error
Missing focus indication

ISSUE DESCRIPTION
When form controls are displaying an error state, their border color changes to yellow and there is no indication of focus aside from the presence of a blinking caret in text inputs or the change in size of the label for controls that have adopted the "float label" pattern (refer to Inadequate focus indication (QLC-10) ). For controls without these features (such as the <select> input in 58 - Select (error), there is no visible focus indicator.

RECOMMENDATIONS
All focusable elements must have a visible focus indication. For instances where this focus indication is not present or has been suppressed, ensure that an appropriate focus indication is provided.

Nominally, WCAG 2.1 does not define how clear/visible a focus indication needs to be - even relying on subjectively faint focus indications passed the relevant success criterion. However, we strongly suggest implementing visually strong and unambiguous focus indication (such as a thicker outline), which will aid sighted keyboard users in navigating and operating the components.

In the case of form controls displaying an error state, ensure that there is an additional focus indication, in addition to the existing error indication.

**AC**

**Comment by ssabo1** [comment]
Confirm: This is no longer an issue post Orbit 
![image](https://user-images.githubusercontent.com/46761151/107709357-10a25b00-6c93-11eb-80d0-7772fdf19c45.png)

**Comment by ssabo1** [comment]
This has been fixed with Orbit
